content/docs/getting-started/first-tasks/_index.md

```diff
@@ -11,29 +11,29 @@ weight: 200
 
 2. Click on the **Scans** icon. In the Scans window, you can create a new scan configuration.
 
-    <img src="/img/openclarity-ui-2.png" alt="OpenClarity UI Scan" title="OpenClarity UI Scan" />
+    <img src="/img/openclarity-ui-2.png" alt="OpenClarity UI Scan" title="OpenClarity UI Scan" width="800px" />
```

Comment:
From looking at the preview, the images look too big

Reply:
not the fullscreen ones, but the detail (partial screen) ones

Reply:
Here is a screenshot about how it is going to look like, wdyt?
![Screenshot 2024-08-26 at 14 52 53](https://github.com/user-attachments/assets/c4c5e8d3-28b7-40d1-b50b-f3b0869b28b5)

Reply:
I think the size of the first (full screen) looks fine but the second is too big

Reply:
How about this?
![Screenshot 2024-08-26 at 15 57 11](https://github.com/user-attachments/assets/a8a5bdac-0650-41e5-8252-f2b973d90159)

Reply:
neat 👍  maybe centralised? though not sure if it will look better

Reply:
The problem with centralizing is that the page is much wider e.g. on my screen and the image would be much further to the right it would be a stretch for the eye 🙂